crawl-ref/source/files.cc

```diff
@@ -2085,6 +2091,53 @@ bool get_save_version(reader &file, int &major, int &minor)
     return true;
 }
 
+static bool _convert_obsolete_species()
+{
+    // At this point the character has been loaded but not resaved, but the grid, lua, stashes, etc have not been.
+#if TAG_MAJOR_VERSION == 34
+    if (you.species == SP_LAVA_ORC)
+    {
+        if (!yes_or_no("This <red>Lava Orc</red> save game cannot be loaded as-is. If you "
+                       "load it now, your character will be converted to a Hill Orc. Continue?"))
+        {
+            you.save->abort(); // don't even rewrite the header
+            delete you.save;
+            you.save = 0;
```

Comment:
Leaving this for future investigation since it is used in a bunch of other places in files.cc.